ti/savings/types.go

```diff
@@ -0,0 +1,17 @@
+package savings
+
+import "github.com/harness/ti-client/types"
+
+type CacheType int32
+
+const (
+	GRADLE CacheType = 1
+)
+
+type Overview struct {
```

Comment:
Where is this being used?

Reply:
It'll be used in the feature. Check CI-10763